subfun/BIDS_FFX.m

```diff
@@ -18,23 +18,28 @@ function BIDS_FFX(action, degreeOfSmoothing, opt)
     fprintf('opt.mat file loaded \n\n')
 end
 
-if opt.isMVPA
+if nargin<4
+    isMVPA = 0;
+end
+
+% Suffix definition (MVPA or not)
+if isMVPA
     mvpaSuffix = 'MVPA_';
 else
     mvpaSuffix = '';
 end
 
-
 %%
 % load the subjects/Groups information and the task name
 [group, opt, BIDS] = getData(opt);
 
 % creates prefix to look for
-if opt.isMVPA
-    [prefix, motionRegressorPrefix] = getPrefix('MVPA', opt, degreeOfSmoothing);
-else
-    [prefix, motionRegressorPrefix] = getPrefix('FFX', opt, degreeOfSmoothing);
-end
+% if opt.isMVPA
```

Comment:
I suspect we are keeping this commented her as a reminder that we want to work on potentially using differently preprocessed files for the MVPA (to be discussed and decided after the workshop).